Beta_Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.py

```diff
@@ -0,0 +1,328 @@
+        data.append(analyzer)
+    data = json.loads(json.dumps(data, cls=DatetimeEncoder))
+
+    ec = {'AWS.AccessAnalyzer.Analyzers(val.arn === obj.arn)': data}
+    human_readable = tableToMarkdown("AWS Access Analyzer Analyzers", data)
+    return_outputs(human_readable, ec)
+
+
+def list_analyzed_resource(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn')
+    }
+
+    if args.get('maxResults'):
+        kwargs['maxResults'] = int(args.get('maxResults'))
+    if args.get('resourceType'):
+        kwargs['resourceType'] = args.get('resourceType')
+
+    response = client.list_analyzed_resources(**kwargs)
+    data = []
+    for resource in response['analyzedResources']:
+        resource['analyzerArn'] = args.get('analyzerArn')
+        data.append(resource)
+
+    ec = {'AWS.AccessAnalyzer.Analyzers(val.analyzerArn === obj.analyzerArn).Resource(val.resourceArn === obj.resourceArn)': data}
+    human_readable = tableToMarkdown("AWS Access Analyzer Resource", data)
+    return_outputs(human_readable, ec)
+
+
+def list_findings(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn')
+    }
+
+    if args.get('maxResults'):
+        kwargs['maxResults'] = int(args.get('maxResults'))
+
+    filters = {}
+    if args.get('resourceType'):
+        filters['resourceType'] = {"eq": [args.get('resourceType')]}
+
+    if args.get('status'):
+        filters['status'] = {"eq": [args.get('status')]}
+
+    if len(filters) > 0:
+        kwargs['filter'] = filters
+
+    response = client.list_findings(**kwargs)
+    data = json.loads(json.dumps(response['findings'], cls=DatetimeEncoder))
+
+    ec = {'AWS.AccessAnalyzer.Findings(val.id === obj.id)': data}
+    human_readable = tableToMarkdown("AWS Access Analyzer Findings", data)
+    return_outputs(human_readable, ec)
+
+
+def get_analyzed_resource(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn'),
+        'resourceArn': args.get('resourceArn')
+    }
+
+    response = client.get_analyzed_resource(**kwargs)
+    data = json.loads(json.dumps(response['resource'], cls=DatetimeEncoder))
+    data['analyzerArn'] = args.get('analyzerArn')
+
+    ec = {'AWS.AccessAnalyzer.Analyzers(val.analyzerArn === obj.analyzerArn).Resource(val.resourceArn === obj.resourceArn)': data}
```

Comment:
are you sure about this DT? dose it merge properly?
isn't `AWS.AccessAnalyzer.Analyzers.Resource(val.resourceArn === obj.resourceArn)` enough ?